routes/routes.go

```diff
@@ -0,0 +1,20 @@
+package routes
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/gror/controllers"
+)
+
+func CreateRoute() {
+
+	r := mux.NewRouter()
+
+	r.HandleFunc("/docker/config", controllers.CreateDocker).Methods("POST")
+	r.HandleFunc("/docker/config/{id}", controllers.GetDocker).Methods("GET")
+	r.HandleFunc("/docker/config/{id}", controllers.UpdateDocker).Methods("PUT")
+	log.Fatal(http.ListenAndServe(":7800", r))
```

Comment:
this method should contain only routes, all other code like starting server and initializing NewRouter should be in your main file